### ISSUE TITLE
Alarm gets armed away by itself after restarting home assistant

### ISSUE THREAD

**Issue by flyman1664** [nielsfaber/alarmo#463]
### Alarmo Version

v1.9.3

### HA Version

2022.5.2

### Bug description

When restarting home assistant, alarm is armed away.
I have to disable it each time.

![image](https://user-images.githubusercontent.com/49916159/167318238-c43d6b39-9213-4f85-b7cf-3e87fbeae690.png)

### Steps to reproduce

On home assistant's restart

**Comment by nielsfaber** [comment]
Alarmo should only start in armed Away mode in case it was in that state before HA was stopped. 
Could there be an automation which arms Alarmo?

You can enable debug logging for alarmo by adding this to the configuration.yaml of HA and restarting HA:
```yaml
logger:
  default: warning
  logs:
    custom_components.alarmo: debug
```

The logs for Alarmo will show up in the normal HA logs.
Could you share the result of the logs around the time of restarting HA?

**Comment by flyman1664** [comment]
Well, I haven't reproduced the issue until now, I don't get it. I will try further tests to find out how to reproduce this behavior.

**Comment by jprokopowich** [comment]
I am seeing the same behavior

**Comment by RACAT3** [comment]
This seems to have been fixed in Home Assistant Core 2022.6.1

**Comment by nielsfaber** [comment]
The behaviour of alarmo during restarting of HA has been _drastically_ changed in version v1.9.4.
The alarm should always start up in the state prior to restarting HA, I expect the problem cannot occur anymore.

Please confirm the issue is resolved by updating your version of alarmo (feel free to leave it running for a couple of weeks before drawing conclusions). Please close the issue if the problem does not occur for you anymore.
